fixtures/small/aref_field_expected.rb

```diff
@@ -1,3 +1,47 @@
 # As above
 a[0] = b
 # So below
+
+a[puts(a)] = ""
+a[puts(a)] = ""
+a[()] = ""
+a[
+  class A
+    def foo
+      puts(a)
+    end
+  end
+] = ""
+a[
+  begin
+    ""
+  rescue StandardException
+    ""
+  end
+] = ""
```

Comment:
I'm gonna call this a "reasonable" rendering? I mean, users writing this is already pretty surprising, so I think this is the most consistent way to handle it rather than special-casing aref fields to inline these or something like that 🤷‍♂️